packages/config-webpack/src/plugins/InlineManifestPlugin.ts

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-param-reassign, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, import/no-extraneous-dependencies */
+/* eslint-disable no-param-reassign, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
```

Comment:
Why did you change the lints?

Reply:
Eslint reported `Unused eslint-disable directive (no problems were reported from 'import/no-extraneous-dependencies').` 

But I will probably revert it.